examples/customOpenAPIschema.md

```diff
@@ -0,0 +1,230 @@
+# Using a Custom OpenAPI schema
+
+A kustomization file supports adding your own
+OpenAPI schema to define merge keys and patch
+strategy.
+
+Make a place to work:
+
+<!-- @placeToWork @testAgainstLatestRelease -->
+```
+DEMO_HOME=$(mktemp -d)
+```
+
+We'll be editing our own CRD:
```

Comment:
s/CRD/[custom resource]/

it's a custom resource, not a custom resource definition.



```
[custom resource]: https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/
```

Reply:
done